website/docs/table-group/data-table/data-table-a11y.md

```diff
@@ -3,20 +3,55 @@ title: DataTable
 tabs: Design('data-table'), Example('data-table-code'), API('data-table-api'), A11y('data-table-a11y'),  Changelog('data-table-changelog')
 ---
 
+## What component has
+
+### Keyboard support
+
+Table: Keyboard support
+
+| Key                                                   | Function                                              |
+| ----------------------------------------------------- | ----------------------------------------------------- |
+| `Tab`                                                 | Moves focus to the next focusable element.            |
+| `Shift + Tab`                                         | Moves focus to the previous focusable element.        |
+| `Left Arrow`, `Right Arrow`, `Up Arrow`, `Down Arrow` | Move the focus across the grid of table cells.        |
+| `Enter`                                               | Activates interactive elements inside the table grid. |
+
+### Roles & attributes
+
+The following list describes roles and attributes that component already has.
+
+| Component / element               | Role / Attribute                        | Usage                                                                                                                                                                                                                                                                                                             |
+| --------------------------------- | --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DataTable`                       | `grid`                                  | Identifies the element that serves as the grid widget container. The `grid` element isn't focusable because it implements the practice described in the section on [Managing Focus Within Components Using a Roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex). |
```

Comment:
```suggestion
| `DataTable`                       | `role="grid"`                                  | Identifies the element that serves as the grid widget container. The `grid` element isn't focusable because it implements the practice described in the section on [Managing Focus Within Components Using a Roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex). |
```
I think it might be better to format roles the same way as attributes, adding the "role=" part, because someone not very familiar with the topic might think that it's an attribute called "grid" (same about the roles below)

Reply:
Oh! I was also thinking about it, but totally forgot to discuss. Agree! 😎